indico/modules/events/contributions/models/subcontributions.py

```diff
@@ -131,6 +131,14 @@ def timetable_entry(self):
         """Convenience property so all event entities have it"""
         return self.contribution.timetable_entry
 
+    @property
```

Comment:
I would have the same `@property` as in `Contribution` ([like this])(https://github.com/indico/indico/blob/devel/indico/modules/events/contributions/models/contributions.py#L252).

This makes me think that `Contribution` and `Subcontribution` can probably use some mixin since they are not so far apart.